testsuite/MDAnalysisTests/analysis/test_leaflet.py

```diff
@@ -42,3 +44,11 @@ def test_leaflet_finder(self):
         assert_equal(bottom_heads.indices, np.arange(2521,4670,12), err_msg="Found wrong leaflet lipids")
 
 
+    def test_string_vs_atomgroup_proper(self):
+        from MDAnalysis.analysis.leaflet import LeafletFinder
+        lfls_ag = LeafletFinder(self.universe, self.lipid_heads, pbc=True)
+        lfls_string = LeafletFinder(self.universe, self.lipid_head_string, pbc=True)
+        groups_ag = lfls_ag.groups()
+        groups_string = lfls_string.groups()
+        assert_equal(groups_string[0].indices, groups_ag[0].indices)
+        assert_equal(groups_string[1].indices, groups_ag[1].indices)
```

Comment:
My point was more about making sure that the group where actually the sames in the two cases (with an AtomGroup and with a selection string). A simple `assert_equal(len(groups_ag), len(group_string))` would make sure that the selection string does not spawn an unexpected leaflet.